android-components/components/feature/webcompat/src/main/assets/extensions/webcompat/manifest.json

```diff
@@ -2,8 +2,13 @@
   "manifest_version": 2,
   "name": "Mozilla Android Components - Web Compatibility Interventions",
   "description": "Urgent post-release fixes for web compatibility.",
+<<<<<<< HEAD
   "version": "106.0.0",
   "applications": {
+=======
+  "version": "108.1.0",
+  "browser_specific_settings": {
```

Comment:
I'm not sure what to do in this case?

Reply:
Should we want to keep this property as it's? 

```json
  "browser_specific_settings": {
    "gecko": {
      "id": "webcompat@mozilla.org",
      "strict_min_version": "102.0"
    }
```

Reply:
We can go with either, so feel free to keep the "old" one.

Reply:
Confirm with Tom via slack, that it's ok to user either, as both are supported in 107. This is now changing to the new way for the MV3 project for addons, and we've apparently supported both ways for a while).